### PR TITLE
snapper: add pt_BR translation

### PR DESCRIPTION
**Pull request by isaacvicente**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**
- `snapper 0.10.3`

### CODE REVIEW

File: pages.pt_BR/linux/snapper.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# snapper
+
+> Utilitário que cria, compara e reverte snapshots com suporte a linhas de tempo de snapshots automáticos.
+> Mais informações: <https://en.opensuse.org/Portal:Snapper>.
```
Comment (kbdharun) [inline pages.pt_BR/linux/snapper.md:4]:
```suggestion
> Mais informações: <http://snapper.io/manpages/snapper.html>.
```

**Review by kbdharun — CHANGES_REQUESTED** [review]
Hi @isaacvicente, Welcome to tldr. Can you refer to the main (English) page <https://github.com/tldr-pages/tldr/blob/main/pages/linux/snapper.md> and retranslate this page again (Try to not change the commands in translations)?

File: pages.pt_BR/linux/snapper.md (hunk 1 above)
Comment (isaacvicente) [reply pages.pt_BR/linux/snapper.md:4]: Yes, for sure. Sorry about not reading with attention all the recommendations :)

**Comment by kbdharun** [comment]
Continued in #8645

**Pull request closed without merging**